packages/gql/codegen.ts

```diff
@@ -59,6 +60,24 @@ export const SUBGRAPHS = {
       return this.endpoints()[network];
     },
   },
+  [Subgraph.BalancerInternalManager]: {
+    name: Subgraph.BalancerInternalManager,
+    endpoints() {
+      const baseEndpoint = "https://api.thegraph.com/subgraphs/name/bleu-llc";
```

Comment:
`#would` leave a comment mentioning this is a fork of the pools subgraph that's to be merged to Balancer's own subgraph